databases/week1/hw-week1.sql

```diff
@@ -0,0 +1,47 @@
+
+select * from task ;
+-- task 1
+select count(*) as tasks_count from task;
+
+-- task 2
+select title from task
+where due_date is null;
+
+ -- task 3
+select title from task
+where status_id = 3;
+
+-- task 4
+select title from task
+where status_id = 1 and 2;
+
+
+-- task 5
+select * from task
+order by created desc;
+
+-- task 6
+select title from task
+order by created desc;
+
+ -- task 7
+ select title,due_date from task
+ where description like '%database%' 
+ or title like '%database%';
+ 
+ -- task 8
+select task.title, status.name from task
+join status on task.status_id = status_id;
```

Comment:
for joining two tables we need an id from table A and reference to the column in table B, for example:
 SELECT * FROM A JOIN B ON A.id = B.a_id;
you can take a look at JOIN clause here (https://www.mysqltutorial.org/mysql-join/)

Reply:
yes it should be 
join status on task.status_id = status.id;